Src/BootCamp.Chapter/LogUtility/Logger.cs

```diff
@@ -0,0 +1,37 @@
+﻿using BootCamp.Chapter.LogUtility;
+using System.IO;
+
+namespace BootCamp.Chapter
+{
+    public class Logger
+    {
+        private const string LogPath = @"..\..\..\Output\Log.txt";
+
+        public Logger()
+        {
+            File.Delete(LogPath);
+        }
+
+        private const int ConsoleLogger = 1;
+        private const int FileLogger = 2;
+        private readonly int LogOption = ConsoleLogger; //Set to either ConsoleLogger or FileLogger
```

Comment:
Make this a method and set the logger itself based on the option.

Reply:
I have coded change based on what I think you are saying to have a method to return back which log option to use.  I created GetLogOption method to do this.